test/EndToEnd/ProjectBuildTests.cs

```diff
@@ -443,7 +443,7 @@ private static string DetectExpectedDefaultFramework(string template = "")
                        || template.StartsWith("nunit")
```

Comment:
the 'proper' way to fix this would be to remove the various `StartsWith` checks (aka removing this entire chunk of logic) since this chunk was used to act as a tracking list of what templates still needed to update. With this change, the list is empty!

Reply:
@baronfel many thanks for the suggestion, went ahead and removed it.